### ISSUE TITLE
citgm yargs failing - Unnecessary escape character

### ISSUE THREAD

**Issue by GeorgeAdams95** [yargs/yargs#721]
I am seeing the following error running `citgm yargs`

```
# /tmp/3ecc36a5-7c79-414c-8a98-18e8257ea4a0/yargs/test/usage.js:2334:45: Unnecessary escape character: \].
# /tmp/3ecc36a5-7c79-414c-8a98-18e8257ea4a0/yargs/test/usage.js:2351:45: Unnecessary escape character: \].
# /tmp/3ecc36a5-7c79-414c-8a98-18e8257ea4a0/yargs/test/yargs.js:453:124: Unnecessary escape character: \].
# /tmp/3ecc36a5-7c79-414c-8a98-18e8257ea4a0/yargs/test/yargs.js:453:164: Unnecessary escape character: \].
# /tmp/3ecc36a5-7c79-414c-8a98-18e8257ea4a0/yargs/test/yargs.js:812:47: Unnecessary escape character: \].
# /tmp/3ecc36a5-7c79-414c-8a98-18e8257ea4a0/yargs/test/yargs.js:851:60: Unnecessary escape character: \].
```
any help would be appreciated

**Comment by bcoe** [comment]
@GeorgeAdams95 this is fixed on master; it's the standard linting library enforcing a new rule, with regard to regexes -- I'll get a new release of yargs out as soon as possible with the fix, can you build against master to test?

**Comment by GeorgeAdams95** [comment]
yes I have, looks good, please let me know when you have cut a new release and published to npm

**Comment by bcoe** [comment]
@GeorgeAdams95 this should now be fixed in `yargs@6.5.0`, let me know if you run into any issues 👍
